doc/whats_new.rst

```diff
@@ -1,14 +1,60 @@
-0.1.5
-=======
+0.2
+===
 
 Changelog
 ---------
 
 The new minimum required version of scikit-learn is 0.13
 
-- Fix to control the behaviour of cut_coords or number of cuts in plot_stat_map.
-  For consistency, number of cuts is changed to default value 7.
-
+New features
+............
+   - The new module :mod:`nilearn.connectome`.
+   - The new class :class:`nilearn.connectome.ConnectivityMeasure` in
+     :mod:`nilearn.connectome` has tools for computing functional
+     connectivity matrices.
+   - The new function :func:`nilearn.connectome.sym_to_vec` in
+     :mod:`nilearn.connectome` is implemented as a helper function to
+     :class:`nilearn.connectome.ConnectivityMeasure`.
+   - The new class :class:`nilearn.decomposition.DictLearning` in
+     :mod:`nilearn.decomposition` is an algorithm implemented to decompose
+     resting state functional connectivity datasets in a data driven manner.
```

Comment:
I would say `A decomposition method similar to ICA that imposes sparsity on components instead of independance between them.`